starter/script.js

```diff
@@ -1 +1,56 @@
+        if (scareAimRun.toUpperCase() === "SCARE") {
+          console.log("A surge of adrenaline runs through you as you fire the pistol into the air and scream.");
+          var scream = prompt("What do you yell?");
+          if (scream.length > 10) {
+            console.log("You scream " + "'" + scream + "' " + "at the top of your lungs and fire the pistol a second time. The slurping ceases immediately and you listen intently for what seems like an eternity. After a couple hours pass you run out of your house and never return. You move to a new town, far away in a distant county in hope of finding peace but it isn’t long before the terrible sound returns. You spend the last few months of your life moving from town to town before being driven mad. Like Smith before you, you are never seen alive again.")
+          } else {
+            console.log("You scream " + "'" + scream + "' " + "as loud as you can but the creature's pace only quickens. Overcome with terror you drop the gun and fall to your knees. A massive shadow creeps up the rug in front of you before blocking out all the light in the room. As you try to reach for the gun your vision fails you and you drop to the floor.");
+          }
+        } else if (scareAimRun.toUpperCase() === "AIM") {
+          console.log("You wait, feeling confident and in control of the situation. Suddenly the wet, slurping sound ceases. After thirty seconds you think about peeking around the corner to see if the creature is still there but you wonder if you should wait a little longer.")
+          var minutes = prompt("How many minutes do you decide to wait?");
+          if (parseInt(minutes) >= 30) {
+            console.log("After " + minutes + " minutes pass you run out of your house and never return. You move to a new town, far away in distant county in hope of finding peace but it isn’t long before the terrible sound returns. You spend the last few months of your life moving from town to town before being driven mad. Like Smith before you, you are never seen alive again.")
+          } else {
+            console.log("You wait " + minutes + " minutes then turn the corner and walk directly into the creature. Trying to make sense of the creature’s countless heads and limbs is an impossibility. You do everything you can to raise the gun and fire but your vision quickly fails you and you drop to the floor. You are never heard from again.");
+          }
+        } else if (scareAimRun.toUpperCase() === "RUN") {
+          console.log("You run toward the back door and realize your senses must have deceived as you find the horrifying creature waiting for you in the hallway. Trying to make sense of the creature’s many limbs and irregular shape is an impossibility. You do everything you can to raise the gun and fire but your vision quickly fails you and you drop to the floor. You are never heard from again.");
+        }
```

Comment:
Encourage you to include an `else` statement in case the user does not provide proper input.